figures/filters.py

```diff
@@ -12,6 +12,7 @@
 )
 from student.models import CourseEnrollment
 
+from figures.pipeline.course_daily_metrics import get_enrolled_in_exclude_admins
```

Comment:
Ideally we want a different home module for this function. I'd put it in sites, but the course id is what drives the filtering. I'll likely revisit this when I start looking into object level permissions